media/css/firefox/new/break-free.scss

```diff
@@ -0,0 +1,193 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../pebbles/components/buttons-download';
+@import '../../pebbles/components/footer-simple';
+@import '../../pebbles/components/platform-footer-links';
+
+#outer-wrapper {
+    background: #0c99d5;
+}
+
+#tabzilla:before {
+    background-color: #4a4a4a;
+}
+
+#masthead {
+    @include at2x('/media/img/firefox/new/break-free/pattern-halftone.png');
+    @include background-size(20px 223px);
+    background-color: #ffcd02;
+    background-repeat: repeat-x;
+    padding-bottom: 10px;
+    position: relative;
+    text-align: center;
+
+    @media #{$mq-high-res} {
+        @include background-size(20px 223px);
+    }
+
+    h1 {
+        clear: right;
+        padding-bottom: 20px;
+        position: relative;
+
+        &:before {
+            @include at2x('/media/img/firefox/template/logo-large.png');
+            @include background-size(108px 112px);
+            background-repeat: no-repeat;
+            content: '';
+            display: block;
+            height: 108px;
+            margin: 0 auto 5px;
+            width: 112px;
+        }
+
+        @media #{$mq-high-res} {
+            &:before {
+                @include background-size(108px 112px);
+            }
+        }
+    }
+
+    &:after {
+        background: #fff;
+        bottom: 0;
+        content: '';
+        display: block;
+        height: 20px;
+        left: 0;
+        position: absolute;
+        right: 0;
+        width: 100%;
+        visibility: visible;
+        border-top: 2px solid #000;
+        border-bottom: 2px solid #000;
+    }
+}
+
+@supports (transform: skewY(-1deg)) {
+    @media screen and (max-width: 1400px) {
+        #masthead:after {
+            @include transform-origin(100%);
+            @include transform(skewY(-1deg));
+        }
+    }
+}
+
+.main-content {
+    text-align: center;
+    padding: 45px 20px 100px;
+
+    .download-link.button.button-green {
```

Comment:
Here also. Maybe we do need this level of specificity since our buttons are kind of a mess.